Directory.Build.props

```diff
@@ -3,7 +3,7 @@
     <IsPackable>false</IsPackable>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateDocumentationFile>false</GenerateDocumentationFile>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>latest</LangVersion>
```

Comment:
is it not safer/more manageable to specify `10.0` explicitly?

Reply:
If I remember right, there was a change in how "latest" was interpreted by different VS versions. I ran into that issue lately and had to explicitly define the lang version.

Reply:
Fixed, 10.0